src/price/crypto-price-box.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { SFC } from 'react';
+import injectSheet from 'react-jss';
+
+import { StyleSheet, StyledComponentProps } from '../react-jss.types';
+
+import { PriceSummary } from './price-summary';
+
+const styles: StyleSheet = {
+    cryptoPriceBox: {
+        height: 'calc(100% - 36px)',
+        padding: '18px 0 18px 0',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        justifyContent: 'flex-start',
+        marginRight: '40px',
+        marginLeft: '40px',
+    }
+};
+
+export type CryptoPriceBoxProps = {
+    locale: string,
+    cryptoCurrency: string,
+    cryptoValue: string,
+    toCurrency: string
+    toValue: string
+}
+
+const CryptoPriceBoxComponent : SFC<StyledComponentProps & CryptoPriceBoxProps> = ({classes, children, locale, cryptoCurrency, cryptoValue, toCurrency, toValue }) => (
+    <div className={classes.cryptoPriceBox}>
+        <PriceSummary locale='' currency={cryptoCurrency} value={cryptoValue}/>
```

Comment:
This should still receive a locale, since they are also used for formatting non-currency numbers which can behave *very* differently depending on region. https://blogs.msdn.microsoft.com/oldnewthing/20060417-06/?p=31513

Reply:
@dfcowell good catch, so here we need to change the format to not be currency or remove the symbol though. right? cheers